web/cypress/integration/route/data-loader-import.spec.js

```diff
@@ -0,0 +1,198 @@
+    cy.contains('api101_mm_customer').should('be.visible');
+    cy.contains('api101_mm_customer/{customer_id}').should('be.visible');
+    cy.contains('api101_mm_customers').should('be.visible');
+
+    // remove route
+    for (let i = 0; i < 3; i += 1) {
+      cy.get(selector.listTbody).get(selector.listRow).contains('More').click();
+      cy.contains('Delete').should('be.visible').click();
+      cy.contains('OK').should('be.visible').click();
+      cy.get(selector.notification).should('contain', data.deleteRouteSuccess);
+      cy.get(selector.notificationCloseIcon).click();
+    }
+  });
+
+  it('should import API101 with duplicate upstream', () => {
+    cy.visit('/');
+    cy.contains('Route').click();
+
+    cy.get(selector.refresh).click();
+    cy.contains('Advanced').click();
+    cy.contains('Import').should('be.visible').click();
+
+    // select Data Loader type
+    cy.contains('OpenAPI 3').should('be.visible').click();
+    cy.get(selector.selectDropdown).contains('OpenAPI 3').should('be.visible').click();
+    cy.get(selector.drawer).get(selector.task_name).type('api101_mm');
+    cy.get(selector.fileSelector).attachFile(cases.API101);
+    cy.get(selector.drawer).contains('Submit').click();
+    cy.get(selector.drawer).contains('Import Failed').should('be.visible');
+    cy.get(selector.drawer).contains('Total 1 upstream imported, 1 failed').click();
+    cy.get(selector.drawer).contains('key: api101_mm is conflicted').should('be.visible');
+    cy.get(selector.drawer).contains('Close').click();
+
+    // remove route
+    for (let i = 0; i < 3; i += 1) {
+      cy.get(selector.listTbody).get(selector.listRow).contains('More').click();
+      cy.contains('Delete').should('be.visible').click();
+      cy.contains('OK').should('be.visible').click();
+      cy.get(selector.notification).should('contain', data.deleteRouteSuccess);
+      cy.get(selector.notificationCloseIcon).click();
+    }
+  });
+
+  it('should import API101 with non-merge mode', () => {
+    cy.visit('/');
+    cy.contains('Route').click();
+
+    cy.get(selector.refresh).click();
+    cy.contains('Advanced').click();
+    cy.contains('Import').should('be.visible').click();
+
+    // select Data Loader type
+    cy.contains('OpenAPI 3').should('be.visible').click();
+    cy.get(selector.selectDropdown).contains('OpenAPI 3').should('be.visible').click();
+    cy.get(selector.drawer).get(selector.task_name).type('api101_nmm');
+    cy.get(selector.drawer).get(selector.merge_method).click();
+    cy.get(selector.fileSelector).attachFile(cases.API101);
+    cy.get(selector.drawer).contains('Submit').click();
+    cy.get(selector.drawer).contains('Import Successfully').should('be.visible');
+    cy.get(selector.drawer).contains('Total 5 route imported, 0 failed').click();
+    cy.get(selector.drawer).contains('Close').click();
+
+    // check result
+    cy.get(selector.listTbody).get(selector.listRow).should('have.length', 5);
+
+    // remove route
+    /**/
```

Comment:
This section seems to be able to be deleted

Reply:
removed